src/alto-ui/SideNav/SideNav.scss

```diff
@@ -348,7 +348,8 @@ $nav-btn-hover-text-dark: $white;
 .sidenav__panel {
   position: absolute;
   background: $white;
-  box-shadow: 1px 0 0 $coolgrey-20;
+  // border-right: 1px solid $coolgrey-20;
```

Comment:
why the comment?

Reply:
I forgot to remove it. It's done now. Thanks